client/rest/test/db/CatapultDb_spec.js

```diff
@@ -401,7 +411,45 @@ describe('catapult db', () => {
 				];
 
 				// Act + Assert:
-				return runTestAndVerifyIds(dbBlocks, db => db.blocks(undefined, account1.address, paginationOptions), [10]);
+				return runTestAndVerifyIds(dbBlocks, db =>
+					db.blocks(undefined, account1.address, undefined, undefined, paginationOptions), [10]);
+			});
+
+			it('fromTimestamp', () => {
+				// Arrange:
+				const dbBlocks = [
+					createBlock(10, 1, account1.publicKey, account1.address, 23456),
+					createBlock(20, 1, account1.publicKey, account1.address, 12345)
```

Comment:
for this test, would prefer if you add a third block with height > 23456

test is currently testing:
1. == fromTimestamp included
2. < fromTimestamp not included

but is missing
3. > fromTimestamp included

and then make similar changes to other NEW tests

Reply:
I added some tests!